src/bptl/camunda/tasks.py

```diff
@@ -79,6 +79,9 @@ def task_execute_and_complete(fetched_task_id):
         logger.warning("Task %r has been already run", fetched_task_id)
         return
 
+    instance_id = fetched_task.instance_id
+    logger.info("Task is part of process instance %d", instance_id)
```

Comment:
```suggestion
    logger.info("Task is part of process instance %s", instance_id)
```
-> task instance ID is typically a UUID